src/controllers/DomainsController.ts

```diff
@@ -58,10 +63,19 @@ export class DomainsController {
     };
 
     domainName = domainName.toLowerCase();
-
     const supportedTLD = isSupportedTLD(domainName);
 
     if (!supportedTLD) {
+      track({
```

Comment:
We can probably make it a function and call it whenever we need to avoid duplication

```suggestion
      const trackEvent = () => track({ ... });
      
      trackEvent();
```